tribeiro/MTAOS/AddAberration.ipynb

```diff
@@ -0,0 +1,896 @@
+{
```

Comment:
<p>Vertical coma --&gt; shouldn't [:, 3:4] be vertical coma?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>